server/chat-commands/moderation.js

```diff
@@ -1411,11 +1413,11 @@ exports.commands = {
 				userid,
 				...Object.keys(targetUser.prevNames),
 				...targetUser.getAltUsers(true).map(user => user.getLastId()),
-			]);
+			], lineCount);
```

Comment:
imo this should probably not work with hidealtstext.

Reply:
That's fair.